lib/cutting_edge/app.rb

```diff
@@ -60,8 +61,10 @@ class App < Sinatra::Base
       repo_defined?(source, org, name)
       @name = name
       @svg = url("/#{source}/#{org}/#{name}/svg")
+      @md = "[![Cutting Edge Dependency Status](#{@svg})](#{url("/#{source}/#{org}/#{name}/info")})"
```

Comment:
Does `url()` take account of the out-facing server url?

May be nice to add a label to the embedded Markdown like so:

`"[![Cutting Edge Dependency Status](#{@svg} \"CuttingEdge Dependency Status\")](#{url("/#{source}/#{org}/#{name}/info")})"`

Reply:
> Does url() take account of the out-facing server url?

Should do: ["It takes reverse proxies and Rack routers into account, if present."](http://sinatrarb.com/intro.html)